helper/staking/staking.go

```diff
@@ -106,18 +126,20 @@ var (
2610de8565b9750610e61836110cb565b925050600181019050610e42565b5085935050505092915050565b610e8581611196565b82525050565b6000610e986044836110e9565b9150610ea38261128a565b606082019050919050565b6000610ebb601d836110e9565b9150610ec6826112ff565b602082019050919050565b6000610ede6027836110e9565b9150610ee982611328565b604082019050919050565b6000610f016012836110e9565b9150610f0c82611377565b602082019050919050565b6000610f24601a836110e9565b9150610f2f826113a0565b602082019050919050565b610f43816111a2565b82525050565b610f52816111de565b82525050565b610f61816111e8565b82525050565b6000602082019050610f7c6000830184610e0f565b92915050565b60006020820190508181036000830152610f9c8184610e1e565b905092915050565b6000602082019050610fb96000830184610e7c565b92915050565b60006020820190508181036000830152610fd881610e8b565b9050919050565b60006020820190508181036000830152610ff881610eae565b9050919050565b6000602082019050818103600083015261101881610ed1565b9050919050565b6000602082019050818103600083015261103881610ef4565b9050919050565b6000602082019050818103600083015261105881610f17565b9050919050565b60006020820190506110746000830184610f3a565b92915050565b600060208201905061108f6000830184610f49565b92915050565b60006020820190506110aa6000830184610f58565b92915050565b6000819050602082019050919050565b600081519050919050565b6000602082019050919050565b600082825260208201905092915050565b600082825260208201905092915050565b6000611105826111de565b9150611110836111de565b9250827fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff03821115611145576111446111f8565b5b828201905092915050565b600061115b826111de565b9150611166836111de565b925082821015611179576111786111f8565b5b828203905092915050565b600061118f826111be565b9050919050565b60008115159050919050565b60006fffffffffffffffffffffffffffffffff82169050919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b6000819050919050565b600063ffffffff82169050919050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b600080fd5b7f4e756d626572206f662076616c696461746f72732063616e2774206265206c6560008201527f7373207468616e204d696e696d756d52657175697265644e756d56616c69646160208201527f746f727300000000000000000000000000000000000000000000000000000000604082015250565b7f4f6e6c79207374616b65722063616e2063616c6c2066756e6374696f6e000000600082015250565b7f56616c696461746f72207365742068617320726561636865642066756c6c206360008201527f6170616369747900000000000000000000000000000000000000000000000000602082015250565b7f696e646578206f7574206f662072616e67650000000000000000000000000000600082015250565b7f4f6e6c7920454f412063616e2063616c6c2066756e6374696f6e000000000000600082015250565b6113d281611184565b81146113dd57600080fd5b50565b6113e9816111de565b81146113f457600080fd5b5056fea26469706673582212204c2964a8ac13edacf7e87d19aef0b89ce42603271129d8494f5203b3ed954c6964736f6c63430008070033"
```

Comment:
Should we maybe put these bytecode in to a separate .go file ?

Reply:
In my opinion no, it is only used for staking inside staking.go. But if you disagree we can